spring-cloud-azure-config/src/main/java/com/microsoft/azure/ConfigHttpClient.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure;
```

Comment:
Package name should be `com.microsoft.azure.spring.cloud.azure.config`?

Reply:
Updated to `com.microsoft.azure.config`